src/error.rs

```diff
@@ -1,11 +1,144 @@
-///! Module contains error handling and error types for laminar
-mod error_kinds;
-mod network_error;
-
-pub use self::error_kinds::{FragmentErrorKind, PacketErrorKind};
-pub use self::network_error::ErrorKind;
-
-use std::result;
+///! This module contains the laminar error handling logic.
```

Comment:
two nits: this should be `//!`

second, we should have a space between the module doc and the first line of code in it. Right now it makes it look like you are adding a doc comment for `use crate...`

Reply:
ow, must have slipped through.